drake/doc/ubuntu.rst

```diff
@@ -30,13 +30,18 @@ Install the prerequisites::
     # Ubuntu 14.04 LTS (Trusty)
     sudo apt-get install cmake-curses-gui
 
+    # These are not required for the basic "make" rule.  We suggest that you
```

Comment:
I agree.  I would like to try to improve these install instructions to be kinder to users, but was content with this as incremental progress.  Do you think it's worth calling out more items as optional, or removing this new comment until we do a broader refactor of the install?